### PR TITLE
default getenv

### PR DESCRIPTION
**Pull request by leonardoalt**

### CODE REVIEW

**Review by pacheco — COMMENTED** [review]
lgtm for the time being

File: riscv-runtime/src/std_support.rs [hunk 1]
```diff
@@ -58,7 +58,7 @@ extern "C" fn sys_getenv(
     _varname: *const u8,
     _varname_len: usize,
 ) -> usize {
-    todo!()
+    0
```
Comment (pacheco) [inline riscv-runtime/src/std_support.rs:61]: a small comment here would be nice though, saying this is implementing the empty env

Comment (lvella) [reply riscv-runtime/src/std_support.rs:61]: I'm fine after this fix.

Comment (leonardoalt) [reply riscv-runtime/src/std_support.rs:63]: updated

**Review by pacheco — APPROVED** [review]

**Pull request merged**